src/App.js

```diff
@@ -1,23 +1,107 @@
-import logo from './logo.svg';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import Modal from 'react-modal';
```

Comment:
Interesting - I did not know about Modal from react-modal! Something for me to look into.